pkg/combustion/registry.go

```diff
@@ -249,89 +236,6 @@ func (c *Combustion) configureEmbeddedArtifactRegistry(ctx *image.Context) (bool
 	return true, nil
```

Comment:
If there are no images we don't send back and error but we use the bool so that we don't continue configuring the EAR
```
configured, err := c.configureEmbeddedArtifactRegistry(ctx)
if err != nil {
	log.AuditComponentFailed(registryComponentName)
	return nil, fmt.Errorf("configuring embedded artifact registry: %w", err)
}

if !configured {
	log.AuditComponentSkipped(registryComponentName)
	zap.S().Info("Skipping embedded artifact registry since the provided manifests/helm charts contain no images")
	return nil, nil
}
```